plugins/wazuh-core/server/services/initialization/routes.ts

```diff
@@ -0,0 +1,242 @@
+import { schema } from '@osd/config-schema';
```

Comment:
You could separate the routes in several files

Reply:
For now, I will keep the route definitions in the same file because they are few.